build-system/global-configs/canary-config.json

```diff
@@ -33,7 +33,6 @@
   "expUnconditionedAdxIdentity": 0.01,
   "expUnconditionedDfpIdentity": 0.01,
   "expUnconditionedCanonicalHoldback": 0.01,
-  "font-display-swap": 1,
```

Comment:
Should this get removed like you're doing, or should it be changed to 0?

Since I'm not fully familiar with the difference, I'll let the others approve :)

Reply:
@rsimha either way works. just less bytes to deliver this way